src/game.js

```diff
@@ -0,0 +1,37 @@
+class Game {
+
+  constructor(frame = new Frame()) {
+    this.score = 0;
+    // this.frames = Array(10).fill(frame);
+    // this.frames = [frame, frame, frame, frame, frame, frame, frame, frame, frame, frame];
+    this.frames = [new Frame(), new Frame(), new Frame(), new Frame(), new Frame(), new Frame(), new Frame(), new Frame(), new Frame(), new Frame()];
```

Comment:
I'm not sure of the answer, but is it better to instantiate all 10 frames at the outset? Or at each go?